examples/11.Encoder/11.Encoder.ino

```diff
@@ -6,8 +6,6 @@ Dragster robot(MMAX_16_OHM);
 
 void setup() {
   pinMode(ENC1, INPUT);
-  robot.begin();
-  robot.drive(120, 0);
```

Comment:
Why you remove robot.begin and .drive? A sample will not work.

Reply:
We changed the chapter itself. It's not convenient to play with the encoder and watch the changes in port monitor while the wheels are spinning. We added the directive to spin it manually.

Reply:
Understand. OK.